apps/src/javalab/Playground.js

```diff
@@ -197,4 +243,8 @@ export default class Playground {
     this.isGameRunning = false;
```

Comment:
Comment for backgroundElement (gitHub won't let me comment that high): It feels a bit odd that we're editing backgroundElement with DOM manipulation but everything else with React. Would it make sense to also move that to React (possibly as a follow-up item)?

Reply:
We could do this as a follow-up, since it's only one element it felt less necessary. We also have the same situation for sound.

Reply:
👍